libandroid-navigation/src/main/java/com/mapbox/services/android/navigation/v5/internal/navigation/MapboxNavigationNotification.kt

```diff
@@ -93,28 +93,31 @@ internal class MapboxNavigationNotification : NavigationNotification {
 
     // Package private (no modifier) for testing purposes
     fun generateArrivalTime(routeProgress: RouteProgress, time: Calendar): String? =
-            ifNonNull(mapboxNavigation, routeProgress.currentLegProgress()) { mapboxNavigation, currentLegProgress ->
-                val options = mapboxNavigation.options()
-                val legDurationRemaining = currentLegProgress.durationRemaining()
-                val timeFormatType = options.timeFormatType()
-                val arrivalTime =
-                        formatTime(time, legDurationRemaining, timeFormatType, isTwentyFourHourFormat)
-                String.format(etaFormat, arrivalTime)
-            }
+        ifNonNull(
+            mapboxNavigation,
+            routeProgress.currentLegProgress
+        ) { mapboxNavigation, currentLegProgress ->
+            val options = mapboxNavigation.options()
+            val legDurationRemaining = currentLegProgress.durationRemaining()
+            val timeFormatType = options.timeFormatType()
+            val arrivalTime =
+                formatTime(time, legDurationRemaining, timeFormatType, isTwentyFourHourFormat)
+            String.format(etaFormat, arrivalTime)
+        }
```

Comment:
WDYT? Does it looks better or not?
```suggestion
            ifNonNull(
                mapboxNavigation?.options(), 
                routeProgress.currentLegProgress().durationRemaining()
            ) { options, legDurationRemaining  ->
                val arrivalTime = formatTime(
                    time, 
                    legDurationRemaining, 
                    options.timeFormatType(), 
                    isTwentyFourHourFormat
                )
                String.format(etaFormat, arrivalTime)
            }
```